filenamify.js

```diff
@@ -1,10 +1,9 @@
-import trimRepeated from 'trim-repeated';
 import filenameReservedRegex, {windowsReservedNameRegex} from 'filename-reserved-regex';
-import stripOuter from 'strip-outer';
 
 // Doesn't make sense to have longer filenames
 const MAX_FILENAME_LENGTH = 100;
 
+const reRepeatedReservedChars = /([<>:"/\\|?*\u0000-\u001F]){2,}/g; // eslint-disable-line no-control-regex
```

Comment:
You did not fully address the feedback in https://github.com/sindresorhus/filenamify/pull/12#discussion_r379363483

Reply:
I didn't move it because i saw that `reControlChars` is also a `g` regex, and it's outside `filenamify()`. Should I move that too?